hazelcast/include/hazelcast/client/spi/DefaultObjectNamespace.h

```diff
@@ -20,6 +20,7 @@
 #define HAZELCAST_CLIENT_SPI_DEFAULTOBJECTNAMESPACE_H_
 
 #include <string>
+#include <functional>
```

Comment:
why?

Reply:
As far as I remember, there was a compile error using the Visual Studio 2015 compiler under Windows without this include. Not sure if this is required anymore.